### ISSUE TITLE
Error: no such tag defined in the VCF header: FORMAT/DP ' returned non-zero exit status 255

### ISSUE THREAD

**Issue by kevjp** [chapmanb/bcbio-nextgen#1034]
I re-ran the pipeline as instructed in #1031 and the pipeline ran fine until it hit this error. I suspect I need to update something given that this issue stems from the upgrade I issued in #1026 
```
Traceback (most recent call last):
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/provenance/do.py", line 21, in run
    _do_run(cmd, checks, log_stdout)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/provenance/do.py", line 95, in _do_run
    raise subprocess.CalledProcessError(exitcode, error_msg)
CalledProcessError: Command 'set -o pipefail; bcftools query -f '[%GC][\t%DP][\t%SAMPLE]\n' -R /home/kevin/Documents/NGS_Pipeline/INDELanalysis/QXT_InDel/input/bedprep/8_2015-09-16_qxtindel1-sort-callable-callableblocks.bed QXT-InsDel-Pool8-QXT-InDel_S8_L001_with-gc.vcf.gz >> tx/tmpdqGetY/QXT-InsDel-Pool8-QXT-InDel_S8_L001_gc-depth-parse.tsv
Warning: Assuming INFO/GC
Error: no such tag defined in the VCF header: FORMAT/DP
' returned non-zero exit status 255
Traceback (most recent call last):
  File "/usr/local/bin/bcbio_nextgen.py", line 226, in <module>
    main(**kwargs)
  File "/usr/local/bin/bcbio_nextgen.py", line 43, in main
    run_main(**kwargs)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 37, in run_main
    fc_dir, run_info_yaml)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 80, in _run_toplevel
    for xs in pipeline.run(config, run_info_yaml, parallel, dirs, samples):
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 201, in run
    samples = qcsummary.report_summary(samples, run_parallel)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/qcsummary.py", line 1037, in report_summary
    samples = run_parallel("coverage_report", samples)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multi.py", line 28, in run_parallel
    return run_multicore(fn, items, config, parallel=parallel)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multi.py", line 86, in run_multicore
    for data in joblib.Parallel(parallel["num_jobs"])(joblib.delayed(fn)(x) for x in items):
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 657, in __call__
    self.dispatch(function, args, kwargs)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 404, in dispatch
    job = ImmediateApply(func, args, kwargs)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 142, in __init__
    self.results = func(*args, **kwargs)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/utils.py", line 50, in wrapper
    return apply(f, *args, **kwargs)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multitasks.py", line 98, in coverage_report
    return qcsummary.coverage_report(*args)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/qcsummary.py", line 1055, in coverage_report
    data = cov.variants(data)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/variation/coverage_experimental.py", line 212, in variants
    "Calculating GC content and depth for %s" % in_vcf)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/provenance/do.py", line 21, in run
    _do_run(cmd, checks, log_stdout)
  File "/usr/local/share/bcbio/anaconda/lib/python2.7/site-packages/bcbio/provenance/do.py", line 95, in _do_run
    raise subprocess.CalledProcessError(exitcode, error_msg)
subprocess.CalledProcessError: Command 'set -o pipefail; bcftools query -f '[%GC][\t%DP][\t%SAMPLE]\n' -R /home/kevin/Documents/NGS_Pipeline/INDELanalysis/QXT_InDel/input/bedprep/8_2015-09-16_qxtindel1-sort-callable-callableblocks.bed QXT-InsDel-Pool8-QXT-InDel_S8_L001_with-gc.vcf.gz >> tx/tmpdqGetY/QXT-InsDel-Pool8-QXT-InDel_S8_L001_gc-depth-parse.tsv
Warning: Assuming INFO/GC
Error: no such tag defined in the VCF header: FORMAT/DP
' returned non-zero exit status 255
```
Appreciate any help you can give me.

Thanks
Kevin

**Comment by chapmanb** [comment]
Kevin;
Sorry about the issue. This is some new coverage reporting that @roryk and @lpantano have been working on. Rory and Lorena, would it be possible to only run this in the QC steps as an opt-in with a configuration variable until we get a chance to test it more widely? It looks like it runs by default right now:

https://github.com/chapmanb/bcbio-nextgen/blob/a108a7d583be46b8e66191e6ab6a90165184eb0d/bcbio/pipeline/qcsummary.py#L1047

**Comment by lpantano** [comment]
Hi Kevin, sorry about that.

I am changing this right now. Just to confirm, you don't have `coverage:` inside your yaml file right?

**Comment by kevjp** [comment]
No I don't have coverage in my yaml file. But I will want to have coverage in the future for a specific bed file but it is not critical now. Thanks for looking into this @lpantano.

**Comment by lpantano** [comment]
Thanks, 

I push a fix to skip this if there is no` coverage` tag (ae2f9f75a). Could you tell me what caller did you use, because I will need to fix that if the vcf file doesn't have the `DP` field there, and it didn't happen until now.

thanks a lot for helping with that and sorry about the problem.

**Comment by kevjp** [comment]
Thank you Lorena I am generating a combined ensemble call set using platypus, GATK haplotype caller and samtools.

**Comment by lpantano** [comment]
Thanks. I tried to fix(b37a48fa90) that and shouldn't be a problem, but let's us know if you have a problem  when you try coverage.
